addons/product/models/product_attribute.py

```diff
@@ -14,7 +14,7 @@ class ProductAttribute(models.Model):
     # `_sort_key_attribute_value` in `product.template`
     _order = 'sequence, id'
 
-    name = fields.Char('Attribute', required=True, translate=True)
+    name = fields.Char('Attribute', required=True, translate=True, index=True)
```

Comment:
As the field is translatable, is the index really efficient?
As the index is a BTree index, do you use startswith or equal comparison?